pkgs/development/interpreters/python/mk-python-derivation.nix

```diff
@@ -176,6 +176,8 @@ let
       # default to python's platforms
       platforms = python.meta.platforms;
       isBuildPythonPackage = python.meta.platforms;
+    } // lib.optionalAttrs (attrs?pname) {
```

Comment:
```suggestion
    } // lib.optionalAttrs (attrs ? pname) {
```

Reply:
I prefered to keep it consistent with the rest.